doc/source/getting_started/install_pymapdl.rst

```diff
@@ -0,0 +1,84 @@
+
+
+.. _ref_pymapdl_installation:
+
+***************
+Install PyMAPDL
+***************
+
+Python module
+~~~~~~~~~~~~~
+The ``ansys.mapdl.core`` package currently supports Python 3.8 through
+Python 3.11 on Windows, Mac OS, and Linux.
+
+Install the latest release from `PyPi <pymapdl_pypi_>`_ with:
+
+.. code:: console
+
+   pip install ansys-mapdl-core
+
+Alternatively, install the latest from 
+`PyMAPDL GitHub <pymapdl_issues_>`_ via:
+
+.. code:: console
+
+   pip install git+https://github.com/ansys/pymapdl.git
+
+
+For a local *development* version, install with:
+
+.. code:: console
+
+   git clone https://github.com/ansys/pymapdl.git
+   cd pymapdl
+   pip install -e .
+
+This allows you to install the ``ansys-mapdl-core`` module
+and modify it locally and have the changes reflected in your setup
+after restarting the Python kernel.
+
+
+Offline installation
+~~~~~~~~~~~~~~~~~~~~
+If you lack an internet connection on your install machine, the recommended way
+of installing PyMAPDL is downloading the wheelhouse archive from the 
+`Releases Page <pymapdl_releases_>`_ for your corresponding
+machine architecture.
+
+Each wheelhouse archive contains all the Python wheels necessary to install
+PyMAPDL from scratch on Windows and Linux for Python. You can install
```

Comment:
Is the "for Python" necessary since we note earlier in the sentence that the wheels are Python? It seems redundant.

Reply:
You are right. As always.